MatrixSDK/JSONModels/MXEvent.h

```diff
@@ -157,12 +158,47 @@ FOUNDATION_EXPORT uint64_t const kMXUndefinedTimestamp;
 @property (nonatomic) NSString *sender;
 
 /**
- The event content.
+ The string event (decrypted, if necessary) type as provided by the home server.
```

Comment:
I would say:
"The decrypted event type provided by the home server (string format).
Unlike 'eventType', this field is always filled even for custom events.

@discussion In case of encrypted event, this field is encrypted until the event is decrypted correctly (see [MXSession decryptEvent]).